server/index.js

```diff
@@ -35,16 +37,16 @@ app.use(urlencoded({ extended: true }));
 
 // require("./middleware/auth")({ app, db, pgp });
 import { auth } from "./middleware/auth.mjs";
-auth({ app, db, pgp });
+auth({ app, db, pgp, AS });
```

Comment:
Is there a reason why you're setting this as a parameter instead of importing `express-async-handler` in each page? I don't see any advantages with this approach.